### PR TITLE
Fix resource generation target

### PR DESCRIPTION
**Pull request by vinay-gopalan**
# Overview
Builds on #211 and fixes remaining broken resources.

# Related Issues/Pull Requests
PR #211

### CODE REVIEW

File: plugin/iamutil/internal/generate_resources.go [hunk 1]
```diff
@@ -53,7 +65,7 @@ func main() {
 
 func checkResource(name string, fullPath string, resource *discovery.RestResource, doc *discovery.RestDescription, docMeta *discovery.DirectoryListItems, config iamutil.GeneratedResources) error {
 	for rName, child := range resource.Resources {
-		checkResource(rName, fullPath+"/"+rName, &child, doc, docMeta, config)
+		return checkResource(rName, fullPath+"/"+rName, &child, doc, docMeta, config)
```
Comment (eh-steve) [inline plugin/iamutil/internal/generate_resources.go:68]:
```suggestion
		err := checkResource(rName, fullPath+"/"+rName, &child, doc, docMeta, config)
		if err != nil {
			return err
		}
```

**Comment by eh-steve** [comment]
Hi @vinay-gopalan would you mind merging #218 into this?

**Comment by bmaximuml** [comment]
> Hi @vinay-gopalan would you mind merging #218 into this?

@tomhjp is this something you could review? Thanks

**Comment by eh-steve** [comment]
Thanks for merging the first PR @vinay-gopalan, but I've since noticed the failing tests so I've opened https://github.com/hashicorp/vault-plugin-secrets-gcp/pull/219 if you want to take a look

**Review by fairclothjm — APPROVED** [review]
LGTM! I deleted the generated file and ran `make generate`.

File: plugin/iamutil/internal/generate_resources.go [hunk 2]
```diff
@@ -269,11 +300,11 @@ func writeConfig(config iamutil.GeneratedResources) error {
...
 	srcBytes, err := format.Source(buf.Bytes())
 	if err != nil {
-		log.Printf("[ERROR] Outputting unformatted src:\n %s\n", string(buf.Bytes()))
-		return errwrap.Wrapf("error formatting generated code: {{err}}", err)
+		log.Printf("[ERROR] Outputting unformatted src:\n %s\n", buf.String())
+		return fmt.Errorf("error formatting generated code: %w", err)
 	}
 
-	dst, err := os.Create(outputFile)
+	dst, err := os.Create(filepath.Join(outputFile))
```
Comment (eh-steve) [inline plugin/iamutil/internal/generate_resources.go:307]:
```suggestion
	dst, err := os.Create(outputFile)
```

**Review by eh-steve — APPROVED** [review]

**Comment by bmaximuml** [comment]
Thanks for merging this so quickly @vinay-gopalan. We'd love to get a new release of this plugin so we can start using it. Can you help us with this?

**Comment by bmaximuml** [comment]
Hi @vinay-gopalan, is there a plan for when the next plugin release will be created? Thanks

**Pull request merged**